Gopkg.toml

```diff
@@ -0,0 +1,61 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/inconshreveable/log15"
+  version = "2.11.0"
+
+[[constraint]]
+  name = "github.com/jessevdk/go-flags"
+  version = "1.2.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/satori/go.uuid"
+
+[[constraint]]
+  name = "gopkg.in/src-d/core-retrieval.v0"
+  source = "https://github.com/src-d/core-retrieval.git"
+
+[[constraint]]
+  branch = "master"
+  name = "gopkg.in/src-d/framework.v0"
+
+[[constraint]]
+  name = "gopkg.in/src-d/go-billy.v4"
+  source = "https://github.com/src-d/go-billy.git"
```

Comment:
We may need to add version information as there is a source:

```
version = "^4.0.0"
```

Reply:
Not need it, just produces complex depende cies

Reply:
Go dep selection was 4.0.2